config/locales/views.es.yml

```
@@ -386,13 +386,20 @@ es:
       empty: 'No se encontraron bases de datos con ese criterio'
     imports:
       create:
-        imported: 'Guiones importados correctamente'
+        title: 'Importar guiones'
+        scripts_imported: 'Guiones importados correctamente'
+        script_imported: 'Importado'
         no_file: 'Debe proporcionar un archivo'
-        fail: 'Se produjo un error importando los guiones'
+        fail: 'No se importaron los guiones, revise los siguientes.'
```

Este texto quedó raro, pondría algo como:
```suggestion
        fail: 'No se importaron los guiones, revise los errores.'
```

---

genial